dist/src/assembly/bin/zeneventserver-create-db

```diff
@@ -53,25 +53,37 @@ class SchemaMigration(object):
         raise NotImplementedError()
 
 class MysqlMigration(SchemaMigration):
-    def _getAdminCommand(self):
+
+    def _getCommand(self):
+        user = self.dbadminuser
```

Comment:
In the code later, the dbadminuser is removed. Should this be self.user/self.pass?